docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md

```diff
@@ -113,3 +113,16 @@ For more information refer to the [Advanced Test Info](AdvancedTestInfo.md) docu
 ## Build and run samples
 
 To build and run the samples, please look [here](/samples/README_Linux.md).
+
+## Determine call path for SGX quote generation in attestation sample
+
+In the attestation sample, you can either take the in-process call path or out-of-process call path to generate evidence of format `OE_FORMAT_UUID_SGX_ECDSA`. If you wish to specify the call path it takes to generate a quote, here is what you can do:
+* To perform in-process quote generation, unset the environment variable `SGX_AESM_ADDR` and ensure that the DCAP library is installed.
+* To perform out-of-process quote generation, set the environment variable `SGX_AESM_ADDR` to any value and ensure that SGX SDK quote-ex Library is installed.
+
+On Linux, if `SGX_AESM_ADDR` is not set, one can run an existing OE app with out-of-process attestation, using `$ SGX_AESM_ADDR=1 <app_name>`.
```

Comment:
This is confusing because it only mentions Linux. What about Windows? Put the answer into the .md file.

Reply:
Same as above.